k8s/observables_deployment.yaml

```diff
@@ -42,5 +42,12 @@ spec:
                 secretKeyRef:
                   name: elasticsearch-password
                   key: password
+          resources:
+            limits:
+              cpu: '1'
+              memory: 3Gi
+            requests:
+              cpu: 250m
+              memory: 1Gi
```

Comment:
I think you can match these with the dashboard resources in gitops-core.

Reply:
Done

Reply:
they're different https://github.com/greymatter-io/gitops-core/blob/main/k8s/outputs/dashboard.cue#L53-L54

Reply:
Coolio